### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by floelhoeffel**

### CODE REVIEW

**Comment by floelhoeffel** [comment]
please wait with review I am adding an image

File: the-example-app/README.md [hunk 1]
```diff
@@ -1,36 +1,31 @@
-What is `The Example App model`?
+Content model: `The Example App`
 ================================
 
 The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
-Contentful. If you want to see the model in action, we do have a [hosted version](https://contentful-example-app.herokuapp.com/), for you
+Contentful. If you want to see the model in action, we do have a [hosted version](https://the-example-app-nodejs.herokuapp.com/), for you
```
Comment (Roblinde) [inline the-example-app/README.md]: remove filler "do"

"If you want to see the model in action, we have..."

File: the-example-app/README.md [hunk 2]
```diff
@@ -1,36 +1,31 @@
...
 to check out.
 
-Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
+Through our [command line app](https://github.com/contentful/contentful-cli) you can import this model into your own space. Doing so will be
```
Comment (Roblinde) [inline the-example-app/README.md]: I would consider removing the second sentence entirely. Just leaving:
"Through our [command line app](https://github.com/contentful/contentful-cli) you can import this model into your own space."

and then following with the command.

File: the-example-app/README.md [hunk 3]
```diff
@@ -1,36 +1,31 @@
...
-Stay tuned for updates on the model and example apps.
-
-
-
-Credentials
+Versions
 ===========
 
-**⚠️WARNING: The following keys are just for demonstration purposes. Normally you would store them on a save place somwhere in the app, but for the demo, feel free to use them :WARNING⚠️**
+We are hosting versions of this content model for demo purposes. `The Example App` can be connected to these versions through `variables.env` when run locally or through query paramters when hosted. For example: https://the-example-app-nodejs.herokuapp.com/?space_id=<YOUR_CLONED_SPACE_ID>&delivery_token=<YOUR_DELIVERY_TOKEN>&preview_token=<YOUR_PREVIEW_TOKEN>
```
Comment (Roblinde) [inline the-example-app/README.md]: paramters -> parameters

Comment (Roblinde) [inline the-example-app/README.md]: I would consider adding "different" as in "We are hosting different versions..." not quite sure if it's necessary, but I feel it reads better.

File: the-example-app/README.md [hunk 4]
```diff
@@ -1,36 +1,31 @@
...
+
+| Version  | Space Id | Content Delivery API - access token | Content Preview API - access token |
+| - | - | - | - |
+| 1.0 | `ft4tkuv7nwl0` | `57459fe48bd2b1bef4855294455af52562dbc0c7f0eb84f8b2cd68692c186417` | `a9972e3cd83528def2fc9d3428c67cd622eb26d0a24239718c6ac61fe0288f2f` |
 
-This example uses the following keys
+**Note:** We are sharing read-only tokens here for demo purposes. Tokens should in general be treated like passwords and not shared with a public audience.
```
Comment (Roblinde) [inline the-example-app/README.md]: I think we need a "be" here, "and not be shared with a public audience."

File: the-example-app/README.md [hunk 5]
```diff
@@ -1,36 +1,31 @@
...
 
-* Space ID: `ft4tkuv7nwl0`
-* Content Delivery API (CDA): `57459fe48bd2b1bef4855294455af52562dbc0c7f0eb84f8b2cd68692c186417`
-* Content Preview API (CPA): `a9972e3cd83528def2fc9d3428c67cd622eb26d0a24239718c6ac61fe0288f2f`
+Visual graph of content model
+=============================
+Using [contentful-graph](https://www.npmjs.com/package/contentful-graph) the following graph was generated to visualize the content model.
```
Comment (Roblinde) [inline the-example-app/README.md]: I would like to reword slightly:

"The following graph was generated using contentful-graph to visualize the content model."

Just to put emphasis on the graph and not the tool used to create it.

**Review by Roblinde — CHANGES_REQUESTED** [review]

**Review by Roblinde — APPROVED** [review]

**Review by mariobodemann — APPROVED** [review]

**Pull request merged**